protocol/src/main/kotlin/website/tbd/tbdex/protocol/Message.kt

```diff
@@ -19,7 +21,7 @@ class MessageMetadata(
   val createdAt: OffsetDateTime
 )
 
-sealed interface MessageData
```

Comment:
This was giving me an error so I just unsealed it... hoping that's acceptable?

Reply:
oh yeah I know what the issue is. fine for now. for sealed classes/interfaces all the implementers need to be in the same file